.storybook/themes/commerce.scss

```diff
@@ -456,6 +456,33 @@
 				),
 			),
 			blocks: (
+				alert-bar: (
+					align-items: center,
+					background-color: var(--background-color-danger),
+					display: flex,
+					flex-wrap: nowrap,
+					justify-content: space-between,
+					padding-left: var(--global-spacing-4),
+					line-height: var(--global-line-height-9),
+					components: (
+						button-default: (
+							background: none,
+							border: none,
+							color: var(--text-color-danger),
+							flex: 0 0 auto,
+						),
+						link: (
+							color: var(--text-color-danger),
+							display: block,
+							font-weight: var(--global-font-weight-7),
+							line-height: var(--global-line-height-2),
+							overflow: hidden,
+							text-decoration: none,
+							text-overflow: ellipsis,
```

Comment:
Is this a new requirement? I know the existing implementation doesn't look too nice with the text but from designs there is not truncation of text - the alert bar would get bigger to allow for the more text. We should bring this up to design and product on the changes proposed

Reply:
now matches V2 and will wrap and expand to fulfill text requirements.